src/components/explore-projects/ExplorePojects.js

```diff
@@ -29,14 +29,14 @@ const ExploreProjects = () => {
   };
   // All images are wrapped in lists and lists are wrapped a container.
 
-  exploreProjectsUtils.handleResponsiveness(
-    matches,
-    matches2,
-    matches3,
-    matches4,
+  exploreProjectsUtils.handleResponsiveness({
+    tablet,
+    betweenTabletAndBigPhones,
```

Comment:
`tabletAndLargeMobile`, but not sure if it's necessary to specify this much.
Desktop, Tablet, Mobile are enough i guess :)